config/default.json

```diff
@@ -147,7 +147,7 @@
       "twoStageCaching": true,
       "optimizeShoppingCart": true,
       "category": {
-        "includeFields": [ "id", "*.children_data.id", "*.id", "children_count", "sku", "name", "is_active", "parent_id", "level", "url_key", "url_path", "product_count", "path"],
+        "includeFields": [ "id", "*.children_data.id", "*.id", "children_count", "*sku", "name", "is_active", "parent_id", "level", "url_key", "url_path", "product_count", "path", "*image","*small_image", "configurable_children.sku", "configurable_children.price", "configurable_children.color", "configurable_children.size", "configurable_children.id"],
```

Comment:
we should probably rather use the this one: 

```
      "productListWithChildren": {
        "includeFields": [ "type_id", "sku", "name", "tax_class_id", "special_price", "special_to_date", "special_from_date", "price", "priceInclTax", "originalPriceInclTax", "originalPrice", "specialPriceInclTax", "id", "image", "sale", "new", "configurable_children.image", "configurable_children.sku", "configurable_children.price", "configurable_children.special_price", "configurable_children.priceInclTax", "configurable_children.specialPriceInclTax", "configurable_children.originalPrice", "configurable_children.originalPriceInclTax", "configurable_children.color", "configurable_children.size", "configurable_children.id", "configurable_children.tier_prices", "product_links", "url_path", "url_key", "status", "tier_prices"],
        "excludeFields": [ "description", "sgn", "*.sgn", "msrp_display_actual_price_type", "*.msrp_display_actual_price_type", "required_options"]
      },
```

We need for sure - `special_price`, `original_price` + `*_incl_tax` , `tier_prices` and `product_links` (`product/setupAssociated` requires `product_links`). Let's better copy the `productListWithChildren`

Moreover, the original logic was to use the `entities.productList` when `isServer=true` and `entities.productListWithChildren` when `isServer=false`. It was because in the SSR mode we wanted to limit then umber of fields to be populated in `__INITIAL_STATE__`. However, we can align it to one setting to make it more understandable